src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -4576,7 +4576,9 @@ void CalcZoneMassBalance(EnergyPlusData &state, bool const FirstHVACIteration)
                     Real64 AdjustedTotalReturnMassFlow = 0;
                     // Calculate return air flow rate using mass conservation equation
                     AdjustedTotalReturnMassFlow = max(0.0, TotInletAirMassFlowRate - TotExhaustAirMassFlowRate + ZoneMixingNetAirMassFlowRate);
-                    AdjustedTotalReturnMassFlow = min(AdjustedTotalReturnMassFlow, zoneEquipConfig.AirLoopDesSupply);
+                    if (!state.dataGlobal->DoingSizing) {
+                        AdjustedTotalReturnMassFlow = min(AdjustedTotalReturnMassFlow, zoneEquipConfig.AirLoopDesSupply);
+                    }
```

Comment:
There are actually 3 places that need this change (line 4564). However, I would rather find a place during zone sizing where zoneEquipConfig.AirLoopDesSupply could be set = to zone air mass flow rate for this time step. This would eliminate these conditionals during the simulation. I have not found that place yet.

Reply:
I agree that this needs improvement to allow for multiple air loops serving the same zone.

Reply:
Added issue #10493.